lib/ontologies_linked_data/models/mod/semantic_artefact_distribution.rb

```diff
@@ -1,7 +1,7 @@
 module LinkedData
     module Models
 
-        class SemanticArtefactDistribution < LinkedData::Models::Base
+        class SemanticArtefactDistribution < LinkedData::Models::ModBase
```

Comment:
where do you do SemanticArtefactDistribution.all ? in that place you need also to use HydraPage

Reply:
it's in the semanticArtefact.rb file. i'll change the logic to use hydrapage